Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+                                                                      )(upper_x)
+            
+            interpolated_lower_surface_data = interpolate.CubicSpline(self.reference_data[self.idx_LE:, 0], 
+                                                                      self.reference_data[self.idx_LE:, 1])(lower_x)
+            
+            # Calculate the squared fit errors of the upper and lower surfaces and sum them to obtain the objective function
+            squared_fit_error_upper_surface = np.sum((upper_y - interpolated_upper_surface_data) ** 2)
+            squared_fit_error_lower_surface = np.sum((lower_y - interpolated_lower_surface_data) ** 2)
+
+            squared_fit_error = squared_fit_error_upper_surface + squared_fit_error_lower_surface
+
+            return squared_fit_error
```

Ruff (0.8.2)

Unnecessary assignment to `squared_fit_error` before `return` statement

Remove unnecessary assignment

(RET504)